frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -289,86 +301,112 @@ const Map: React.FC<MapProps> = ({
     }
   };
 
+  const [infoOpen, setInfoOpen] = React.useState(false);
+  const [layersOpen, setLayersOpen] = React.useState(false);
+
   return (
-    <ReactResizeDetector handleWidth>
```

Comment:
I added the context provider to the map, nothing else on the map was changed other than the two slideouts